internal/pkg/get/get.go

```diff
@@ -0,0 +1,77 @@
+package get
+
+import (
+	"fmt"
+
+	"github.com/jbrekelmans/kube-compose/pkg/config"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	clientV1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type getRunner struct {
+	cfg              *config.Config
+	k8sClientset     *kubernetes.Clientset
+	k8sServiceClient clientV1.ServiceInterface
+}
+
+type KubeComposeService struct {
```

Comment:
Rename to `ServiceDetails`, we have about 50 thousand structs that include "service" in their name.